kernel/aster-nix/src/device/mod.rs

```diff
@@ -35,7 +35,7 @@ pub fn init() -> Result<()> {
     #[cfg(feature = "intel_tdx")]
     let tdx_guest = Arc::new(tdxguest::TdxGuest);
     #[cfg(feature = "intel_tdx")]
-    add_node(tdx_guest, "tdx-guest")?;
+    add_node(tdx_guest, "tdx_guest")?;
```

Comment:
This node should be added only if the TDX feature of the CPU is detected at runtime.